crypto/ake/enclave/src/lib.rs

```diff
@@ -27,6 +27,8 @@ use mc_sgx_compat::sync::Mutex;
 use mc_util_from_random::FromRandom;
 use sha2::{Sha256, Sha512};
 
+pub type SealedClientMessage = EnclaveMessage<ClientSession>;
+
```

Comment:
This should be a real type, even if it's identical to `EnclaveMessage<ClientSession>`---it's always an error to try use an `attest::Message`/`EnclaveMessage` directly as  a `SealedClientMessage`.

Reply:
Fixed. I thought Rust type aliases wouldn't allow you to just pass the aliased type, whoops.